tests/store/tracking/test_file_store.py

```diff
@@ -515,12 +516,17 @@ def test_delete_restore_run(self):
         fs = FileStore(self.test_root)
         exp_id = self.experiments[random_int(0, len(self.experiments) - 1)]
         run_id = self.exp_data[exp_id]["runs"][0]
+        _, run_dir = fs._find_run_root(run_id)
         # Should not throw.
         assert fs.get_run(run_id).info.lifecycle_stage == "active"
         fs.delete_run(run_id)
         assert fs.get_run(run_id).info.lifecycle_stage == "deleted"
+        meta = read_yaml(run_dir, FileStore.META_DATA_FILE_NAME)
+        assert "deleted_time" in meta and meta["deleted_time"] is not None
         fs.restore_run(run_id)
         assert fs.get_run(run_id).info.lifecycle_stage == "active"
+        meta = read_yaml(run_dir, FileStore.META_DATA_FILE_NAME)
+        assert "deleted_time" not in meta or meta["deleted_time"] is None
```

Comment:
```suggestion
        assert "deleted_time" not in meta or meta["deleted_time"] is None
```

Do we need `or meta["deleted_time"] is None`?

Reply:
you're right, i've removed the `meta["deleted_time"] is None` check!